app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ] Please replace `:user` with `:author` exactly as you did in the `post.rb` file so kindly fix this issue as well :+1: check below :point_down:  my suggestion :slightly_smiling_face:  

```suggestion
  belongs_to :author, class_name: 'User', foreign_key: :author_id
```